### DISCUSSION TITLE
How to let the search box only search the current page?

### DISCUSSION THREAD

**Discussion by elllllllaaa** [microsoft-search/pnp-modern-search#2951] [Q&A]
Hi, really hope someone could help :( been trying for many days. I am currently using SharePoint to create a page. 

I would like to create a search box for only search the current page, it's a bit like searching keywords which leads you to that place. Example here (but it's not on SharePoint):

![image](https://user-images.githubusercontent.com/132588823/236245612-ac75dacb-a533-449c-bc9b-0b434ca4df01.png)

However, the solutions I found it's always link to other sites or pages.
![image](https://user-images.githubusercontent.com/132588823/236245740-e4bebae4-b5b1-4e12-9db5-32260fd635e5.png)
But I only want the results to be current page. Does anyone know how to do this? Thank you so much!

**Comment by wobba** [comment]
Duplicate of #2950
